arm-network/2017-06-01/swagger/azureWAN.json

```diff
@@ -0,0 +1,2812 @@
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "name": "VirtualWANName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the VirtualWAN being deleted."
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. VirtualWAN deleted."
+                    },
+                    "202": {
+                        "description": "Request received successfully. VirtualWAN deletion is in progress; follow the Location header to poll for final outcome."
+                    },
+                    "204": {
+                        "description": "No VirtualWANs exist by the name provided."
+                    },
+                    "default": {
+                        "description": "*** Error Response: ***\n\n * 400 Invalid request. VirtualWAN still has VirtualHubs, cannot delete."
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroup/{resourceGroupName}/providers/Microsoft.Network/VirtualWANs": {
+            "get": {
+                "operationId": "VirtualWANs_ListByResourceGroup",
+                "x-ms-examples": {
+                    "VirtualWANListByResourceGroup": { "$ref": "../examples/VirtualWANListByResourceGroup.json" }
+                },
+                "description": "Lists all the VirtualWANs in a resource group.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of all the VirtualWANs in the resource group.",
+                        "schema": {
+                            "$ref": "#/definitions/ListVirtualWANsResult"
+                        }
+                    },
+                    "default": {
+                        "description": "*** Error Response: ***\n\n * 400 Invalid request."
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
```

Comment:
Can virtualWANs be moved from one resource group to another or from one subscription to another? If yes, please add support for the Move API